.github/workflows/terraform-plan-apply.yml

```diff
@@ -191,6 +191,7 @@ jobs:
       ARM_CLIENT_SECRET: ${{secrets.TF_ARM_CLIENT_SECRET}}
       ARM_SUBSCRIPTION_ID: f3fba52d-c7db-46f8-9e7a-766ca869972e
       ARM_TENANT_ID: 99804659-431f-48fa-84c1-65c9609de05b
+      UI_VERSION : https://github.com/NDPH-ARTS/mts-trial-ui/releases/latest/download/mts-trial-ui.zip
```

Comment:
Small thing, but maybe call it UI_BUNDLE since it is not really a version...

Reply:
Good point - I will switch this.